DoctorWho.Db/Repositories/DoctorsRepository.cs

```diff
@@ -9,24 +9,25 @@ namespace DoctorWho.Db.Repositories
 {
     public class DoctorsRepository
     {
-        public static void CreateDoctor(int doctorNumber, string doctorName, DateTime birthDate, DateTime firstEpisodeDate, DateTime lastEpisodeDate)
+        public static DoctorsRepository current { get; } = new DoctorsRepository();
```

Comment:
why do you use a static instance of the class inside itself??
we discussed this issue in the EF Core exercise, right?